src/status_im/contexts/chat/messenger/composer/mentions/style.cljs

```diff
@@ -2,29 +2,38 @@
   (:require
     [quo.foundations.colors :as colors]
     [react-native.platform :as platform]
-    [react-native.reanimated :as reanimated]
     [status-im.contexts.chat.messenger.composer.constants :as constants]))
 
-(defn shadow
-  [theme]
-  (if platform/ios?
-    {:shadow-radius  (colors/theme-colors 30 50 theme)
-     :shadow-opacity (colors/theme-colors 0.1 0.7 theme)
-     :shadow-color   colors/neutral-100
-     :shadow-offset  {:width 0 :height (colors/theme-colors 8 12 theme)}}
-    {:elevation 10}))
+(defn- add-shadow
+  [theme styles]
+  (cond-> styles
+    platform/ios? (assoc :shadow-radius (colors/theme-colors 30 50 theme)
+                         :shadow-opacity (colors/theme-colors 0.1 0.7 theme)
+                         :shadow-color colors/neutral-100
+                         :shadow-offset {:width 0 :height (colors/theme-colors 8 12 theme)})
+    :else (assoc :elevation 10)))
```

Comment:
Modified this to use `assoc` instead of `merge` :eyes:

Reply:
Here I think `cond->` is more convoluted to read than a simple `if` macro.

```clojure
(defn- add-shadow
  [theme styles]
  (if platform/ios?
    (assoc styles
           :shadow-radius  (colors/theme-colors 30 50 theme)
           :shadow-opacity (colors/theme-colors 0.1 0.7 theme)
           :shadow-color   colors/neutral-100
           :shadow-offset  {:width 0 :height (colors/theme-colors 8 12 theme)})
    (assoc styles :elevation 10)))
```